packages/studio-be/package.json

```diff
@@ -68,18 +68,14 @@
     "yargs": "^17.0.1",
     "yn": "^2.0.0"
   },
-  "resolutions": {
-    "fstream": ">=1.0.12",
-    "lodash": ">=4.17.19"
-  },
```

Comment:
Why remove those resolutions?

Reply:
Yarn2 use the resolutions on the main package.json, so these are redundant